extensions/adobe/experience/aam-profile.schema.json

```diff
@@ -0,0 +1,41 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/aam-profile",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Adobe Audience Manager Profile Template Mixin",
+  "type": "object",
+  "description": "Adobe Audience Manager Mixin for use with Schemas for Solution data ingestion. Includes the core/standard Profile XDM as well as the other required Core mixins.",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/profile"],
+  "meta:extends": [
+    "https://ns.adobe.com/xdm/context/profile-segmentation",
+    "https://ns.adobe.com/xdm/context/profile-identities-deprecated",
+    "https://ns.adobe.com/xdm/context/device"
```

Comment:
Why are you adding device information to a profile? It was designed for an instance observation?